torch/csrc/jit/script/lexer.h

```diff
@@ -132,7 +134,7 @@ struct SharedParserData {
         {}, // reserve a level for unary not
         {'<', '>', TK_EQ, TK_LE, TK_GE, TK_NE},
         {'+', '-'},
-        {'*', '/'},
+        {'*', '/', TK_MATMUL, TK_POW},
```

Comment:
TK_POW has  higher precedence than `*` (See https://docs.python.org/3/reference/expressions.html), but here you are giving it the same precedence as `*`.

Reply:
TK_POW is right associative, so it needs to be added to `isRightAssociative`